doc/Quick_Start/README.md

```diff
@@ -66,21 +66,25 @@ https://docs.rakwireless.com/Product-Categories/WisBlock/RAK5005-O/Datasheet/#wi
 
 The RAK5005-O board offers several GPIO's on solder pads or on the WisBlock Sensor or WisBlock IO modules. These GPIO's are named IO1 to IO6 and SW1. These GPIO's are connected to GPIO's of the RAK4631 module.    
 **RAK5005-O GPIO mapping to RAK4631 GPIO ports**
- * IO1 <-> P0.17 (Arduino GPIO number 17)
- * IO2 <-> P1.02 (Arduino GPIO number 34)
- * IO3 <-> P0.21 (Arduino GPIO number 21)
- * IO4 <-> P0.04 (Arduino GPIO number 4)
- * IO5 <-> P0.09 (Arduino GPIO number 9)
- * IO6 <-> P0.10 (Arduino GPIO number 10)
- * SW1 <-> P0.01 (Arduino GPIO number 1)
+ * RAK5005-O <->  nRF52840
+ * IO1       <->  P0.17 (Arduino GPIO number 17)
+ * IO2       <->  P1.02 (Arduino GPIO number 34)
+ * IO3       <->  P0.21 (Arduino GPIO number 21)
+ * IO4       <->  P0.04 (Arduino GPIO number 4)
+ * IO5       <->  P0.09 (Arduino GPIO number 9)
+ * IO6       <->  P0.10 (Arduino GPIO number 10)
+ * SW1       <->  P0.01 (Arduino GPIO number 1)
+ * A0        <->  P0.04/AIN2 (Arduino Analog A2
+ * A1        <->  P0.31/AIN7 (Arduino Analog A7
+ * SPI_CS    <->  P0.26 (Arduino GPIO number 26) 
 
 ### WisBlock Core board --- RAK4631
 
 RAK4631 is the WisBlock Core board, because the MCU stay on this board.
 
-<center><img src="../../assets/repo/RAK4631.png" alt="RAK4631" width="30%" /></center>
+<center><img src="../../assets/repo/RAK4631.png" alt="RAK4631" width="75%" /></center>
 
-<center><img src="../../assets/repo/rak4631-connectors.png" alt="rak4631-connectors" width="75%" /></center>
+<center><img src="../../assets/repo/rak4631-connectors.png" alt="rak4631-connectors" /></center>
```

Comment:
The pictures takes too much space. Spaces between pictures must standard, or at least proportionate.